test/transmit.js

```diff
@@ -1330,43 +1353,119 @@ describe('transmission', () => {
             await server.start();
 
             const res = await Wreck.request('GET', 'http://localhost:' + server.info.port);
-            res.on('data', (chunk) => {
+            res.on('data', (chunk) => { });
 
-                if (!destroyed) {
-                    destroyed = true;
-                    res.destroy();
-                }
+            await team.work;
+            await server.stop();
+
+            expect(res.statusCode).to.equal(200);
+        });
+
+        it('destroy() stream when request aborts before stream drains', async () => {
+
+            const server = Hapi.server();
+
+            const team = new Teamwork.Team();
+            const handler = async (request) => {
+
+                clientRequest.abort();
+
+                const stream = new Stream.Readable({
+                    read(size) {
+
+                        const chunk = new Array(size).join('x');
+
+                        setTimeout(() => {
+
+                            this.push(chunk);
+                        }, 10);
+                    },
+                    destroy() {
+
+                        team.attend();
+                    }
+                });
+
+                await Hoek.wait(100);
+                return stream;
+            };
+
+            server.route({ method: 'GET', path: '/', handler });
+
+            await server.start();
+
+            const clientRequest = Http.request({
+                hostname: 'localhost',
+                port: server.info.port,
+                method: 'GET'
             });
 
+            clientRequest.on('error', () => { /* NOP */ });
```

Comment:
All good either way, but one upside of using `Hoek.ignore` is that it also documents that it's an intentional no-op.